app/policies/application_policy.rb

```diff
@@ -0,0 +1,53 @@
+# frozen_string_literal: true
+
+class ApplicationPolicy
+  attr_reader :user, :record
+
+  def initialize(user, record)
+    @user = user
+    @record = record
+  end
+
+  def index?
+    false
+  end
+
+  def show?
+    false
+  end
+
+  def create?
+    false
+  end
+
+  def new?
+    create?
+  end
+
+  def update?
+    false
+  end
+
+  def edit?
+    update?
+  end
+
+  def destroy?
+    false
+  end
```

Comment:
You can write a test for these methods:

```
  context 'given the user is a keyword owner' do
    it 'permits :new, :create, :index' do
      user = Fabricate(:user)
      keyword = Fabricate(:keyword, user: user)

      policy = described_class.new(user, keyword)

      expect(policy).to permit_action(:new)
      expect(policy).to permit_action(:create)
      expect(policy).to permit_action(:index)
      # ... and other method here
    end
  end

  context 'given the user is an anonymous user' do
    it 'does NOT permit :new, :create, :index' do
      anonymous_user = NullUser.new
      keyword = Fabricate(:keyword)

      policy = described_class.new(anonymous_user, weekly_days_off)

      expect(policy).not_to permit_action(:new)
      expect(policy).not_to permit_action(:create)
      expect(policy).not_to permit_action(:index)
      # ... and other method here
    end
  end
```

What is the `NullUser`?
https://thoughtbot.com/blog/rails-refactoring-example-introduce-null-object